server/application/application.proto

```diff
@@ -163,6 +163,7 @@ message ApplicationPodLogsQuery {
 	optional k8s.io.apimachinery.pkg.apis.meta.v1.Time sinceTime = 6;
 	required int64 tailLines = 7 [(gogoproto.nullable) = false];
 	required bool follow = 8 [(gogoproto.nullable) = false];
+	optional int64 maxLines = 9;
```

Comment:
I think `maxLines` duplicates `tailLines` . The `tailLines` is doing the same using Kubernetes API.

Reply:
`maxLines` is slightly different, more like `tailLines` in reverse. I think this is the simplest way to get a "window" of logs with our current API (i.e. only adding one new parameter). `tailLines` does use k8s API, but `maxLines` does a head slice of the logs returned from the k8s API. 

So if you were to request `tailLines: 200, maxLines: 100`, you'd get the 100th most recent log line up to the 200th most recent log line.